src/CoreWCF.Primitives/src/CoreWCF/Description/TypeLoader.cs

```diff
@@ -397,9 +420,9 @@ private IEnumerable<Type> GetKnownTypes(object[] knownTypeAttributes, CustomAttr
             return knownTypes;
         }
 
-        private KeyedByTypeCollection<IOperationBehavior> GetIOperationBehaviorAttributesFromType(OperationDescription opDesc, Type targetIface, Type implType)
+        private KeyedByTypeCollection<IOperation> GetIOperationAttributesFromType<IOperation>(OperationDescription opDesc, Type targetIface, Type implType)
         {
```

Comment:
Generic types generally start with T. One line 425 it looks like there's an interface called IOperation that you are specifying. It makes the code a little harder to read.